docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/StochasticGradientDescent.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.ML;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification
+{
+    public static class StochasticGradientDescent
+    {
+        // In this examples we will use the adult income dataset. The goal is to predict
+        // if a person's income is above $50K or not, based on different pieces of information about that person.
+        // For more details about this dataset, please see https://archive.ics.uci.edu/ml/datasets/adult.
```

Comment:
>https://archive.ics.uci.edu/ml/datasets/adult [](start = 59, length = 45)

Broken link.

Reply:
I think they're just temporarily down. This is prominent link. It's the first google result for "uci adult". 

---
In reply to: [259450232](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259450232) [](ancestors = 259450232)